src/vendor/wordpress/_wordpress.scss

```diff
@@ -111,6 +106,50 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our staggerItems
+ *    utility, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ * 2. Setting a width of 100% prevents resized and centered images from breaking
+ *    layout when placed after a right-aligned image in a post.
+ * 3. Applies an opinionated wide-image setting that allows an image element to
+ *    spill slightly outsides its container at larger viewports.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  &.aligncenter {
+    width: 100%; /* 2 */
+  }
+
+  &.alignfull {
+    left: 50%;
+    margin-left: -50vw;
+    margin-right: -50vw;
+    right: 50%;
+    width: 100vw;
```

Comment:
These styles seem more verbose than the ones I wrote about and that we use on our current site: https://cloudfour.com/thinks/breaking-out-with-viewport-units-and-calc/#flipping-the-script

Reply:
I agree. Thank you. I attempted to address this feedback in 6664d90